pkg/apis/kappctrl/v1alpha1/types_template.go

```diff
@@ -21,6 +22,21 @@ type AppTemplateYtt struct {
 
 type AppTemplateKbld struct{}
 
+type AppTemplateSopsArgs struct {
+	KMSKeys    []string `json:"kmsKeys,omitempty"`
+	GCPKms     []string `json:"gcpKms,omitempty"`
+	AzureKV    []string `json:"azureKV,omitempty"`
+	PGP        []string `json:"pgp,omitempty"`
+	AWSProifle string   `json:"awsProfile,omitempty"`
+	IgnoreMac  bool     `json:"ignoreMac,omitempty"`
+}
+
+type AppTemplateSops struct {
+	Match      string               `json:"match,omitempty"`
+	MergeFiles bool                 `json:"mergeFiles,omitempty"`
+	Args       *AppTemplateSopsArgs `json:"args,omitempty"`
```

Comment:
i think we should inline various AppTemplateSopsArgs fields into AppTemplateSops.